aiida/backends/manager.py

```diff
@@ -121,9 +121,13 @@ def load_backend_environment(self, profile, validate_schema=True):
     def _load_backend_environment(self):
         """Load the backend environment."""
 
-    @abc.abstractmethod
-    def reset_backend_environment(self):
+    @staticmethod
+    def reset_backend_environment():
         """Reset the backend environment."""
+        from aiida.backends.sqlalchemy import ENGINE, SCOPED_SESSION_CLASS
+        if ENGINE is not None:
+            ENGINE.dispose()
+        SCOPED_SESSION_CLASS = None
```

Comment:
I think that if you want to use this as a 'global' variable you need to do instead `from aiida.backends import sqlalchemy` and then `sqlalchemy.ENGINE.dispose()`; probably not critical since you are not resetting it, but just calling a method, but instead it's essential, I think, for the `SCOPED_SESSION_CLASS`

Reply:
Right!